templates/zerver/api/fixtures.json

```diff
@@ -366,5 +366,45 @@
         "code": "BAD_REQUEST",
         "msg": "User not authorized for this query",
         "result": "error"
+    },
+    "zulip-outgoing-webhook-payload": {
+        "data": "@**Outgoing Webhook Test** Zulip is the world\u2019s most productive group chat!",
```

Comment:
Hmm, the documentation suggests the `data` field should have the mention stripped; how did you construct this fixture @eeshangarg?  @roberthoenig FYI.

Reply:
@timabbott: By creating a bot, and then @-mentioning it in a stream message and using RequestBin as the endpoint! :)

Reply:
Just tested this, can confirm that the @-mention does not get stripped for the `data` property.

Reply:
I think it's okay for `data` to remain unstripped. It seems to be a property that can be used to directly retrieve the message sent and not bother with the `message` object. I think a user would expect the full message anyways.  We should update this in this doc tho (it still talks of stripped messages).